lib/Authentication/AuthTokens/ShibAuthToken.php

```diff
@@ -146,7 +146,26 @@ private function getAttributesInitToken(){
             return;
 
         }
-
+        else if($idp == 'https://aai-dev.egi.eu/proxy/saml2/idp/metadata.php'){
```

Comment:
There's no explanation of why there's all this extra code required when everything else just refers to using a new entitlement.

Reply:
This pull request is still a work in progress

Reply:
Yeah, I've just seen #184. 🤦‍♂